### PR TITLE
RHEL 10 network config change

### PR DESCRIPTION
**Pull request by MarckK**

### CODE REVIEW

File: articles/virtual-machines/workloads/redhat/redhat-images.md [hunk 1]
```diff
@@ -193,6 +193,15 @@ We update only the current minor release in a given image family. With the relea
 
 Current policy is to keep all previously published images. We reserve the right to remove images that are known to cause problems of any kind. For example, images with incorrect configurations due to subsequent platform or component updates might be removed. Images that might be removed follow the current Azure Marketplace policy to provide notifications up to 30 days before image removal.
 
+## Network configuration
+
+Current RHEL 8 and RHEL 9 images leverage the network configuration of DHCP on eth0. The default cloud-init network configuration is overridden by setting "apply_network_config: false".
```
Comment (anujkmaurya) [inline articles/virtual-machines/workloads/redhat/redhat-images.md:198]: can we reword it to 
"we are currently overriding the default cloud-init network configuration by setting "apply_network_config: false" to support RHEL 8 and RHEL 9 images and customers."

File: articles/virtual-machines/workloads/redhat/redhat-images.md [hunk 2]
```diff
@@ -193,6 +193,15 @@ We update only the current minor release in a given image family. With the relea
...
+
+> [!NOTE]
+> From RHEL 10 onwards, images will use the default cloud-init configuration that leverages Azure's IMDS endpoint for network configuration.
+>
+> We request that customers plan their VM configuration accordingly from RHEL 10 onwards. 
+
```
Comment (anujkmaurya) [inline articles/virtual-machines/workloads/redhat/redhat-images.md:204]: it's good

**Review by anujkmaurya — CHANGES_REQUESTED** [review]

**Comment by v-dirichards** [comment]
@mathapli 

Can you review the proposed changes?
 
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge.
 
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by anujkmaurya — APPROVED** [review]

**Comment by anujkmaurya** [comment]
#sign-off

**Comment by ShannonLeavitt** [comment]
@cynthn or @mathapli

Can you review these changes and sign off if you approve?

@MicrosoftDocs/public-repo-pr-review-team

**Comment by cynthn** [comment]
#sign-off
